resources/assets/Application.js

```diff
@@ -39,6 +39,9 @@ const Application = () => {
           <Route path="/campaigns/:id/:status">
             <ShowCampaign />
           </Route>
+          <Route path="/groups" exact>
+            <GroupTypeIndex />
+          </Route>
           <Route path="/group-types" exact>
             <GroupTypeIndex />
           </Route>
```

Comment:
Should we just remove this route 🤷

Reply:
I think it's fine to keep since we have a `/group-types/:id`